Manifest.toml

```diff
@@ -1490,11 +1512,18 @@ git-tree-sha1 = "50ccd5ddb00d19392577902f0079267a72c5ab04"
 uuid = "69024149-9ee7-55f6-a4c4-859efe599b68"
 version = "0.3.5"
 
+[[deps.StringManipulation]]
+git-tree-sha1 = "46da2434b41f41ac3594ee9816ce5541c6096123"
+uuid = "892a3eda-7b42-436c-8928-eab12a02cf0e"
+version = "0.3.0"
+
 [[deps.StructArrays]]
-deps = ["Adapt", "DataAPI", "StaticArrays", "Tables"]
-git-tree-sha1 = "9abba8f8fb8458e9adf07c8a2377a070674a24f1"
+deps = ["Adapt", "DataAPI", "StaticArraysCore", "Tables"]
+git-tree-sha1 = "13237798b407150a6d2e2bce5d793d7d9576e99e"
+repo-rev = "master"
+repo-url = "https://github.com/JuliaArrays/StructArrays.jl.git"
 uuid = "09ab397b-f2b6-538f-b94a-2f83cf4a842a"
-version = "0.6.8"
+version = "0.6.13"
```

Comment:
The problem right now is that the current stable version of `StructArrays` (0.6.12) throws an error when broadcasting over `StructArrays` (which happens right now in [line 360](https://github.com/legend-exp/legend-julia-tutorial/blob/529834842d5f5bef188a4651a114de1816c11ed0/legend-julia-software-tutorial.jmd#L360) of the tutorial).

There is a fix on the current `main` of `StructArrays` but we might want to wait until there is an official release of `0.6.13` to make the tutorial work again.